tests/phpunit/tests/media.php

```diff
@@ -1169,7 +1169,7 @@ public function test_remove_image_size() {
 	 * @ticket 26951
 	 */
 	public function test_has_image_size() {
-		add_image_size( 'test-size', 200, 600 );
+		add_image_size( 'test-size', 200, 600, false, true );
 		$this->assertTrue( has_image_size( 'test-size' ) );
 
```

Comment:
Can you add some actual test coverage for the new parameter here? We should test that setting the parameter rightfully affects the additional registered sizes, and we should also make sure that it is taken into account when checking sizes via the `_wp_filter_image_sizes_additional_mime_type_support()` function.

Reply:
I'll add some coverage in `test_add_image_size`

Reply:
Added additional coverage specifically for the new `additional_mimes` parameter:

* Added coverage for both the expected data from `wp_get_additional_image_sizes` with `additional_mimes` true and false
* Added coverage that `_wp_filter_image_sizes_additional_mime_type_support()` also correctly returns the correct size where `additional_mimes` was set to true